python/setup.py

```diff
@@ -18,7 +19,14 @@
 assert os.path.isfile(_cmake_file)
 
 _build_dir = build_dir = os.path.join(_opencc_rootdir, 'build', 'python')
-_libopenccfile = os.path.join(_clib_dir, 'lib', 'libopencc.so')
+_system = platform.system()
+if _system == 'Darwin':
```

Comment:
why is this part duplicated?

Reply:
This part exists in both `opencc/__init__.py` and `setup.py`.
The `setup.py` script is for building/installing the package and generally it is not a good practice to source the package you are building for.

I admit that it is also a bad practice to have duplicated code but duplicated code is probably the greater evil in the long run.

Reply:
Fixed in d0e78d6